lib/recipes/college-physics/bake

```diff
@@ -3,99 +3,17 @@
 # frozen_string_literal: true
 
 require_relative '../recipes_helper'
+require_relative '../college-physics/college_physics_recipe'
 
-recipe = Kitchen::BookRecipe.new(book_short_name: :college_physics) do |doc|
+college_physics = Kitchen::BookRecipe.new(book_short_name: :college_physics) do |doc|
   include Kitchen::Directions
 
   book = doc.book
-  metadata = book.metadata
-
-  book.search('cnx-pi').trash
-
-  BakePreface.v1(book: book)
-  BakeChapterTitle.v1(book: book)
-  BakeChapterIntroductions.v1(book: book)
-
-  BakeUnclassifiedNotes.v1(book: book)
-  BakeAutotitledNotes.v1(book: book, classes: %w[interactive])
-  BakeIframes.v1(outer_element: book)
-  BakeMathInParagraph.v1(book: book)
-
-  # Check Your Understanding exercises: titled unnumbered exercises
-  book.exercises('$[data-element-type="check-understanding"]').each do |exercise|
-    BakeAutotitledExercise.v2(exercise: exercise, title: I18n.t(:'check-understanding'))
-  end
 
   book.chapters.each do |chapter|
     BakeLearningObjectives.v3(chapter: chapter)
```

Comment:
Just a question to make sure: should the Learning Objectives be baked in college-physics recipe but not in college-physics 2e?

Reply:
yes, in 1e they want the recipe to stay as is, but since this code is shared with 2e and 2e doesn't bake learning-objectives, I separated it using the bake file

Reply:
Thank you for clarifying Karina! :relaxed: